src/selectors/url-state.js

```diff
@@ -189,7 +189,7 @@ export const getProfileName: Selector<null | string> = createSelector(
     if (!pathInZipFile) {
       return null;
     }
-    const pathParts = pathInZipFile.split('/');
+    const pathParts = pathInZipFile.split('%');
     return pathParts[pathParts.length - 1];
```

Comment:
Hey,
I think we shouldn't show the full name, but just the last 2 parts (instead of the last 1 part only).
Otherwise it may be too long to display in the interface.